tests/sources/xml/test_ead.py

```diff
@@ -586,15 +586,6 @@ def test_ead_transform_with_invalid_date_and_date_range_omits_dates(caplog):
     assert ("has a later start date than end date: '2001', '1999'") in caplog.text
 
 
-def test_ead_transform_with_multiple_unitid_gets_valid_ids():
-    ead_xml_records = Ead.parse_source_file(
-        "tests/fixtures/ead/ead_record_attribute_and_subfield_variations.xml"
-    )
-    output_record = next(Ead("aspace", ead_xml_records))
-    for identifier in output_record.identifiers:
-        assert identifier.value != "unitid-that-should-not-be-identifier"
-
-
```

Comment:
@ehanson8 Related to https://github.com/MITLibraries/transmogrifier/pull/185#discussion_r1621226563. I removed the test and in its place are the [tests for the new `get_identifiers` field method](https://github.com/MITLibraries/transmogrifier/pull/187/files#diff-4e446208d5381af3faa605c03e15486e64e9af30c4d54e8f40bec821cb87c851R964-R1006). 🤓